src/components/Menu/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import { IMenu } from "./types";
+import menuOptions from "./const";
+//Translate - i18next
+import { useTranslation } from "react-i18next";
+//styles components
+import { StyledDiv, StyledMenu, StyledOptions, StyledUser } from "./styles";
+//images
+import UserPic from "../../assets/img/user-avatar.svg";
+import Close from "../../assets/img/close-x.svg";
+
+const Menu: React.FC<IMenu> = ({ isOpen, handleClick, userName, userType }: IMenu) => {
+    const { t } = useTranslation();
+    return (
+        <StyledMenu isOpen={isOpen}>
+            <StyledDiv>
+                <img alt="" src={UserPic} />
+                <StyledUser>
+                    <h3>{userName}</h3>
+                    <p>{userType}</p>
+                </StyledUser>
+                <button onClick={handleClick}>
```

Comment:
Creo que asi te quitarias el handleClick, y asi podrias hacer que la boton desactive el menu sin tener que pasarle ninguna funcion en las props 

```suggestion
import { useState } from "react";

const Menu: React.FC<IMenu> = ({ isOpen, userName, userType }: IMenu) => {
    const { t } = useTranslation();

    const [IsOpen, setIsOpen] = useState(isOpen);
    return (
        <StyledMenu isOpen={IsOpen}>
            <StyledDiv>
                <img alt="" src={UserPic} />
                <StyledUser>
                    <h3>{userName}</h3>
                    <p>{userType}</p>
                </StyledUser>
                <button onClick={(): void => setIsOpen(!IsOpen)}>

```